### DISCUSSION TITLE
Using dynamic behavior in component alias configurations

### DISCUSSION THREAD

**Discussion by henokteixeira** [vuetifyjs/vuetify#20446] [Q&A]
Hello, I'm using Vuetify and I have a question about using aliases in components. I wanted to know if it's possible for me to make any dynamic changes to the properties defined in the aliases.

For example, creating an alias for a VTextfield component so that it has a behavior focused on passwords. It has two icons: one to enable the display of passwords and another to disable it. The dynamic behavior would be to change the icon and the TextField type between text and password. I know that there is the option of creating a component just for this, which would allow it to be used. However, I wanted to know if it would be possible to configure this component in this way as an alias.

**Comment by KaelWD** [comment]
No, this would require creating a component anyway in order to manage that state. There is https://github.com/vuetifyjs/vuetify/issues/17192 but it won't be stateful.
